src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -765,78 +766,139 @@ function saturation_adjustment(e_int::FT, ρ::FT, q_tot::FT) where {FT<:Real}
   end
 end
 
-
 """
```

Comment:
L. 748, fix reference: 
```suggestion
See also [`saturation_adjustment_q_tot_θ_liq_ice`](@ref).
```

Reply:
Regarding saturation adjustment: I updated the corresponding section in the design document to include a linearized solution for `T_2`. I suggest you implement this, and provide options to either (a) return the linearized `T_2` as the solution without further iteration (should be good enough for what we need), or (b) use this `T_2` as initial guess for secant iterations.